services/galley/src/Galley/Cassandra/Conversation.hs

```diff
@@ -249,60 +251,111 @@ getConversation conv = do
       <*> UnliftIO.wait cdata
   runMaybeT $ conversationGC =<< maybe mzero pure mbConv
 
-getGlobalTeamConversation :: TeamId -> Client (Maybe Conversation)
-getGlobalTeamConversation tid =
-  (\c -> c {convLocalMembers = [], convRemoteMembers = []})
-    <$$> getConversation (globalTeamConv tid)
+getGlobalTeamConversation ::
```

Comment:
Why the duplication? Can't `getGlobalTeamConversation` simply call `getGlobalTeamConversationById`?